### PR TITLE
First draft at description of feature requests vs bugs.

### LINKED ISSUE #1659: Strategy for triaging issues
[Lodash](https://github.com/lodash/lodash) has an interesting strategy for handling their github issues: [here](https://twitter.com/samselikoff/status/991395669016436736)

In short:

- Feature requests are closed issues with `feature` label and `votes needed` label, and a 👍 emoji reaction
- Bugs are open issues tagged with `bug` label

For a project like Moto where there are an endless number of feature requests (every time AWS adds a service/endpoint), this feels very appealing. It would be great for us to be able to separate out the triage work. Establishing an easy way to give prioritization feedback would also be very helpful.

Any downsides? cc @JackDanger @terrycain

### PR DESCRIPTION
**Pull request by spulec**
Closes #1659

### CODE REVIEW

**Comment by spulec** [comment]
I should also note: if we merge something like this, I will close all existing "enhancements" issues with a message pointing at this guide.

**Comment by mikegrima** [comment]
I think that we should mention that in general, people will submit pull-requests for features when they need a feature for their internal tests. One should not assume that the community will contribute the feature.

We encourage users to submit PRs to implement the features in moto whenever a feature is missing or incomplete.

**Comment by spulec** [comment]
Just documenting that we should utilize [github issue templates](https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository) as part of moving forward with this.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
 
 Moto has a [Code of Conduct](https://github.com/spulec/moto/blob/master/CODE_OF_CONDUCT.md), you can expect to be treated with respect at all times when interacting with this project.
 
+## Feature Requests
+
+Because of the large number of feature requests for new services/endpoints that are opened against Moto, we handle triaging of issues slightly differently than most projects. Feature requests should be submitted in the [issue tracker](https://github.com/spulec/moto/issues), with a description of the expected behavior & use case, where they’ll remain closed until sufficient interest, measured by 👍 reactions, has been [shown by the community](https://github.com/spulec/moto/issues?utf8=%E2%9C%93&q=label%3A%22enhancement%22+sort%3Areactions-%2B1-desc+). Before submitting a request, please search for similar ones in the [closed issues](https://github.com/spulec/moto/issues?q=is%3Aissue+is%3Aclosed+label%3Aenhancement).
```
Comment (bblommers) [inline CONTRIBUTING.md]: ```where they’ll remain closed until sufficient interest```
This sounds like they will be reopened at some point - which, if I understand correctly, is not the case?

```Before submitting a request```
I'd reword this to 'Before submitting an issue', as ```request``` on it's own can be confused with a PR

**Comment by bblommers** [comment]
I like the direction this could take moto, even if it could be a bit confusing for new users.
I'd argue that we do need to use labels a bit more to differentiate between:
 - Feature requests
 - Closed (fixed) issues

Does it also make sense to start using labels to differentiate between implementing:
 - new services
 - new methods
 - missing behaviour from an existing method?

Personally, when I submit a PR, I don't always look for the feature that I expect to be the most popular/has the most upvotes. If I only have a few hours to spend on it, I'll look for something that is easy to pick up.
Having labels to differentiate the type of issue (and by extension, the work involved) would be very useful.

I guess we could also expand the ```## Is there a missing feature?```-paragraph to say: Before creating an issue, please search for closed issues, labeled as x/y/z. Makes it hopefully clear for people.
